.github/workflows/continuous-integration-workflow.yml

```diff
@@ -1,6 +1,18 @@
 name: Continous Integration Tests
 on: [push, pull_request]
 jobs:
+  jenkins:
+    name: Jenkins tests
+    runs-on: ubuntu-18.04
+    steps:
+      -
+        name: Checkout repository
+        uses: actions/checkout@v2.0.0
+      -
+        name: Build image
+        working-directory: jenkins/master
+        run: |
+          docker build -t jenkins .
```

Comment:
Do you mind changing this job to `jenkins-master`, and adding another job `jenkins-slave` next to it that tests the slave? It would be really nice to see that at least building the image works.